lib/screens/headways.ex

```diff
@@ -203,4 +219,9 @@ defmodule Screens.Headways do
       {_, false} -> :off_peak
     end
   end
+
+  defp silver_line_route_headway_key("741"), do: "sl_one"
+  defp silver_line_route_headway_key("742"), do: "sl_two"
+  defp silver_line_route_headway_key("743"), do: "sl_three"
```

Comment:
There's also the 746, SLW, which I think you should include. See here: https://api-v3.mbta.com/routes/746